test/expiring-todo-comments.js

```diff
@@ -0,0 +1,173 @@
+import test from 'ava';
+import avaRuleTester from 'eslint-ava-rule-tester';
+import rule from '../rules/expiring-todo-comments';
+
+const ruleTester = avaRuleTester(test, {
+	env: {
+		es6: true
+	}
+});
+
+const expiredTodoError = expirationDate => ({
+	ruleId: 'expiring-todo-comments',
+	message: `There is a TODO that is past due date: ${expirationDate}`
+});
+
+const avoidMultipleDatesError = expirationDates => ({
+	ruleId: 'expiring-todo-comments',
+	message: `Avoid using multiple expiration dates in TODO: ${expirationDates}`
+});
+
+const havePackageError = pkg => ({
+	ruleId: 'expiring-todo-comments',
+	message: `There is a TODO that is deprecated since you installed: ${pkg}`
```

Comment:
The messages should include the actual TODO message too.

```
// TODO [2200-12-12]: This part
```

Reply:
Done